DataDog/datadog-agent
# [jmxfetch] Bump jmxfetch version from 0.42.0 to 0.42.1

### What does this PR do?

Bumps version of `jmxfetch` to 0.42.1. This version contains no user-visible changes in functionality except a different source now for the underlying artifact (sonatype vs bintray).

### Motivation

We use a different pipeline for publishing artifacts now so this change uses the first artifact produced that way

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

- Run jmxfetch check
- Ensure that it works
